src/components/DropZone/DropZone.tsx

```diff
@@ -363,18 +367,25 @@ export class DropZone extends React.Component<CombinedProps, State> {
       return;
     }
 
-    let size = 'extraLarge';
-    const width = this.node.getBoundingClientRect().width;
-
-    if (width < 100) {
-      size = 'small';
-    } else if (width < 160) {
-      size = 'medium';
-    } else if (width < 300) {
-      size = 'large';
+    let height = Size.ExtraLarge;
+    const wrapperHeight = this.node.getBoundingClientRect().height;
+    if (wrapperHeight < Size.Small) {
+      height = Size.Small;
+    } else if (wrapperHeight < Size.Medium) {
+      height = Size.Medium;
+    } else if (wrapperHeight < Size.Large) {
+      height = Size.Large;
```

Comment:
This is fine but you could: 

`let height` and then add the Size.ExtraLarge to Size.ExtraLarge; as oppose to mutating height. Same with width below.